src/main/resources/templates/reactivate/accountReactivated.html

```diff
@@ -4,13 +4,13 @@
 <div class="container">
     <div class="grid-row">
         <div class="govuk-box-highlight">
-            <h1 class="heading-xlarge">Account reactivation complete</h1>
-            <p class="font-medium">Your account has been successfully reactivated</p>
+            <h1 class="heading-xlarge">Account reactivation final step</h1>
```

Comment:
This doesn't read very well - maybe just trim it down to "Account reactivation"?

Reply:
Done